lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -469,9 +500,10 @@ def main():
             conf_file=dict(default=None, type='path'),
             disable_gpg_check=dict(default=False, type='bool'),
             installroot=dict(default='/', type='path'),
+            autoremove=dict(default=True, type='bool'),
```

Comment:
default=True would be a large change in behaviour.  Is there justification for this (like the dnf command line client defaults to autoremove?)  (If the answer is to change this to default False, remember to change documentation as well).